app/surveys/mci/mci_0205.py

```diff
@@ -0,0 +1,353 @@
+questionnaire.introduction = introduction
+
+# Create the group object
+group = Group()
+group.id = "14ba4707-321d-441d-8d21-b8367366e766"
+group.title = ""
+
+questionnaire.add_group(group)
+questionnaire.register(group)
+
+# Create the block
+block = Block()
+block.id = "cd3b74d1-b687-4051-9634-a8f9ce10a27d"
+block.title = "Monthly Business Survey"
+
+group.add_block(block)
+questionnaire.register(block)
+
+# Add the sections
+
+###
+# Section One
+###
+
+s1 = Section()
+s1.id = "f11bc7dc-3940-4c7d-9d0e-faa6acb23eeb"
+s1.title = ""
+s1.description = ""
+
+block.add_section(s1)
+questionnaire.register(s1)
+
+# Date Range question
+q1 = Question()
+q1.id = "6cc86b54-330c-4465-99b2-34cc7073dc2c"
+q1.title = "What are the dates of the sales period you are reporting for?"
+q1.description = "If possible, this should be for the period {exercise.start_date:%-d %B %Y} to {exercise.end_date:%-d %B %Y}."
+q1.type = "DateRange"
+
+s1.add_question(q1)
+questionnaire.register(q1)
+
+q1r1 = Response()
+q1r1.id = "6fd644b0-798e-4a58-a393-a438b32fe637"
+q1r1.code = "11"
+q1r1.label = "From"
+q1r1.guidance = ""
+q1r1.type = "Date"
+q1r1.options = []
+q1r1.mandatory = True
+q1r1.messages = {
+    "MANDATORY": "Please answer before continuing.",
+    "INVALID_DATE": "The date entered is not valid.  Please correct your answer."
+}
+
+q1.add_response(q1r1)
+questionnaire.register(q1r1)
+
+q1r2 = Response()
+q1r2.id = "06a6a4b7-6ce4-4687-879d-3443cd8e2ff0"
+q1r2.code = "12"
+q1r2.label = "To"
+q1r2.guidance = ""
+q1r2.type = "Date"
+q1r2.options = []
+q1r2.mandatory = True
+q1r2.messages = {
+    "MANDATORY": "Please answer before continuing.",
+    "INVALID_DATE": "The date entered is not valid.  Please correct your answer."
+}
+
+q1.add_response(q1r2)
+questionnaire.register(q1r2)
+
+###
+# Section Two
+###
+
+s2 = Section()
+s2.id = "d9d25a21-41b8-42b2-ac7b-fb21b1036d71"
+s2.title = "Commodities - Retail Turnover"
+s2.description = "<p>- You should enter figures for the reporting period stated above<br>- You should round your figures to the nearest (£) pound</p><h4>Include</h4><ul><li>VAT</li><li>Internet Sales</li></ul>"  # NOQA
+
+block.add_section(s2)
+questionnaire.register(s2)
+
+q2 = Question()
+q2.id = "eedb9f9f-2f2e-41f4-9186-ba7110d9e6a4"
+q2.title = ""
+q2.description = ""
+q2.type = "Currency"
+
+s2.add_question(q2)
+questionnaire.register(q2)
+
+q2r = Response()
```

Comment:
Similar to above, this was left out *for now* in order to limit the scope and minimise the impact of other parts of the system.  It should probably be done sooner rather than later though, but perhaps a different PR